src/Sentry/SentryFailedRequestHandler.cs

```diff
@@ -0,0 +1,100 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using Sentry.Protocol;
+
```

Comment:
```suggestion
```

The `System` namespaces are already being imported globally, and I don't think you need `Sentry.Protocol` here.  In general, we try to reduce `using` statements to  the minimum needed to compile.